sorts/bead_sort.py

```diff
@@ -0,0 +1,42 @@
+
+from itertools import zip_longest
```

Comment:
Hey @PedroDCN. Apparently the build is failing as you have imported `zip_longest` and haven't used it. Please remove it.